src/main/java/net/dries007/tfc/objects/te/TEBarrel.java

```diff
@@ -184,9 +186,9 @@ public void update()
         {
             tickCounter++;
 
-            if (tickCounter == 10)
+            if (tickCounter % 10 == 0)
```

Comment:
Why?

Reply:
I liked the sound playing every 4 ticks more then every 5 ticks, so I incresed the tick counter to 20 so it's divisible with 4 and that's to keep that thing at a 10 tick repeat time